.github/workflows/test.yml

```diff
@@ -44,9 +44,3 @@ jobs:
         with:
           test-command: 'echo unit tests already executed'
           coverage-comment: false
-
-      - name: Coveralls Report
-        uses: coverallsapp/github-action@1.1.3
-        if: success() || failure()
-        with:
-          github-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Why remove coveralls?

Reply:
<img width="581" alt="image" src="https://user-images.githubusercontent.com/7122625/218964268-7c2a5c62-2f01-45b7-a844-f0098b47f375.png">

Jest + istanbul-badges-readme provides a more advanced report (from my perspective) and doesn't require additional github action.
I don't have a strong opinion on it, can move back.

Reply:
True, that's nice. But that's for the readme, thus more visible when looking at the project page usually on `main`, no?
It doesn't seem that visible to me to visualize when compared to the PR comment with the changes in the PR we are reviewing

Reply:
Reverted it :)